.github/workflows/zap-scan-test.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:
       - main
+      - development
```

Comment:
You sure you want to run this every time we merge a feature branch, now? Because this is a Pull Request trigger, its going to run against *test* before the corresponding feature is deployed there, for what it's worth.